modules/compliance-operator-cli-installation.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-installation.adoc
+
+[id="installing-compliance-operator-cli_{context}"]
+= Installing the Compliance Operator using the CLI
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Create a namespace object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-compliance
+----
+
+. Apply the file by running:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
```

Comment:
No need to apply the file if we are creating namespace resource using `$ oc create -f namespace.yaml` and step 2 can be merged in step 1